src/frontend/src/common/components/library/data_table/index.tsx

```diff
@@ -138,6 +140,14 @@ function extractPublicIdsFromDataWFailedGenomeRecovery(data: TableItem[]) {
   return failedSampleIds;
 }
 
+function extractPublicIdsWBadQCData(data: Sample[]) {
+  return data
+    .filter(
+      (s) => s.qc_metrics.length > 0 && s.qc_metrics[0].qc_status === "bad"
```

Comment:
I think it would be nice to have a comment about why we're using the first element of the list, is there only one?

Reply:
how hard is it to transform these to camelCase?  I think this is a best practice because it makes the frontend less dependent on the API design.  i.e. if this was renamed to `sample_qc_status` there would be a single place to update `qcStatus = sample_qc_status` rather than having to find all of the locations of `qc_status`

Reply:
yeah there should only be one entry for now, i'll add a comment!